zwave_js_server/model/node.py

```diff
@@ -383,6 +383,16 @@ async def async_refresh_info(self) -> None:
         """Send refreshInfo command to Node."""
         await self.async_send_command("refresh_info", wait_for_result=False)
 
+    async def async_refresh_values(self) -> None:
+        """Send refreshValues command to Node."""
+        await self.async_send_command("refresh_values", wait_for_result=False)
```

Comment:
Should these include `require_schema=4`? The only other place I saw it was `async_poll_value`, so I wasn't sure.